File: Code/BeforeLegends/Assets/Scripts/Utilities/CharacterModelPrefabs.cs
using UnityEngine;
using System.Collections;

public class CharacterModelPrefabs : MonoBehaviour {

    public static GameObject[] prefabs;

    public GameObject[] prefabs_editor;

    public static GameObject[] battlePrefabs;

    public GameObject[] battlePrefabs_editor;

    public static GameObject[] ressourcePrefabs;

    public GameObject[] ressourcePrefabs_editor;

    void Awake(){
	    prefabs = prefabs_editor;
	    battlePrefabs = battlePrefabs_editor;
	    ressourcePrefabs = ressourcePrefabs_editor;
    }
}

File: Code/BeforeLegends/Assets/Scripts/Utilities/CharacterModelPrefabs.js
#pragma strict

static var prefabs : GameObject[];

var prefabs_editor : GameObject[];

static var battlePrefabs : GameObject[];

var battlePrefabs_editor : GameObject[];

static var ressourcePrefabs : GameObject[];

var ressourcePrefabs_editor : GameObject[];

function Awake(){
	prefabs = prefabs_editor;
	battlePrefabs = battlePrefabs_editor;
	ressourcePrefabs = ressourcePrefabs_editor;
}